lms/djangoapps/grades/transformer.py

```diff
@@ -49,6 +50,12 @@ def collect(cls, block_structure):
         """
         block_structure.request_xblock_fields(*cls.FIELDS_TO_COLLECT)
         cls._collect_max_scores(block_structure)
+        collect_unioned_set_field(
+            block_structure=block_structure,
+            transformer=cls,
+            merged_field_name='containing_subsections',
```

Comment:
@jcdyer I had thought about only saving the field on scorable blocks, but I'm not sure how to do that + keep the inherited behavior we have without requiring a second traversal to delete unneeded intermediary values.